### PR TITLE
Update docs typos, minor rephrasing, fix dead links

### PR DESCRIPTION
**Pull request by jeffjennings**
This PR fixes a few typos in the docs, updates dead hyperlinks in the docs, and fixes the last of the `WARNING`s one gets when building the docs (`make html`).

### CODE REVIEW

File: frank/utilities.py [hunk 1]
```diff
@@ -344,9 +344,9 @@ def estimate_weights(u, v, V, nbins=300, log=True, use_median=False):
 
     The estimation is done assuming that the variation in each bin is dominated
     by the noise. This will be true if:
-        1) The source is axi-symmetric,
-        2) The uv-points have been deprojected,
-        3) The bins are not too wide,
+    1) The source is axi-symmetric,
```
Comment (jeffjennings) [inline frank/utilities.py:347]: I only did this because building the docs complains about `unexpected indentation` here.

**Comment by rbooth200** [comment]
If you've verified that all of the new links work, I'm fine with this

**Comment by jeffjennings** [comment]
thanks for reviewing so fast!  I have meticulously clicked each one.

I guess we decided to get rid of `docs/_build` in the repo, which is why they were all dead (they now all point via the directory structure in the docs).

**Comment by rbooth200** [comment]
docs/_build is a temporary working directory so can't be relied on

**Comment by jeffjennings** [comment]
Yeah all the better that the hyperlinks no longer use `docs/_build`

**Pull request merged**